resources/multisite_drupal_standard.make

```diff
@@ -524,6 +524,7 @@ projects[tmgmt][download][branch] = 7.x-1.x
 projects[tmgmt][download][revision] = bd307cb0cdf55a20092f616aeb0c39bd918aef5d
 projects[tmgmt][download][type] = git
 projects[tmgmt][subdir] = contrib
+projects[tmgmt][patch][] = patches/support_for_link_field-2489134-8.patch
```

Comment:
For contributed patches (not customised by us) it is better to use the patch of drupal.org:
``projects[tmgmt][patch][] = https://www.drupal.org/files/issues/support_for_link_field-2489134-9.patch``